uint32 id
string control_task
---
bool success
string message
